code/b-parasite/Makefile

```diff
@@ -1,17 +1,40 @@
+# Supported platforms:
+# * E73_2G4M08S1C (nrf2480, default)
+# * E73_2G4M08S1E (nrf2833) 
+PLATFORM ?= E73_2G4M08S1C 
+
+ifeq ($(PLATFORM), E73_2G4M08S1E)
+	SRC_FILES += \
+		$(SDK_ROOT)/modules/nrfx/mdk/gcc_startup_nrf52833.S \
+  		$(SDK_ROOT)/modules/nrfx/mdk/system_nrf52833.c
+	TARGETS := nrf52833_xxaa
+	CFLAGS += -DNRF52833_XXAA
+	ASMFLAGS += -DNRF52833_XXAA
+  	LINKER_SCRIPT  := b_parasite_gcc_nrf52833.ld
+HEAP_SIZE := 2048

```

Comment:
Would you please indent these two lines and the two ones in the `else` body below?